_posts/2021/2021-02-08-use-enhanced-memoization-for-false-nil-with-defined.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Use Enhanced Memoization for false/nil with defined?"
+description: "Memoizing false & nil results requires more formality"
+layout: article
+category: ruby
+image:
+  base: "2021/use-enhanced-memoization-for-false-nil-with-defined"
+  alt: "true, false & nil word cloud"
+---
+
+One straightforward performance optimization we discussed using was [memoization using the `||=` operator](ruby/memoize-expensive-operations) however this isn't a solution in all cases.
```

Comment:
*we discussed in the last/in a previous issue was... . However, this isn't a _suitable_ solution in all cases.

Reply:
Maybe hint high-level at which sort of cases it won't work for in the intro?